stdlib/3/asyncio/futures.pyi

```diff
@@ -52,5 +52,7 @@ class Future(Awaitable[_T], Iterable[_T]):
     def __await__(self) -> Generator[Any, None, _T]: ...
     @property
     def _loop(self) -> AbstractEventLoop: ...
+    @property
+    def _callbacks(self) -> Sequence[Callable[[_S], Any]]: ...
```

Comment:
```suggestion
    def _callbacks(self: _S) -> List[Callable[[_S], Any]]: ...
```

`_S` has no meaning, unless `self` is annotated. Also, use a concrete class as return type, instead of a protocol.

Reply:
Also, you do have to support the 3.7 change, where apparently it returns a list of callback-context pairs. You can add `if sys.version_info >= (3, 7):` with the new code.